### ISSUE TITLE
JQuery UI Accordion, Slickgrid issue

### ISSUE THREAD

**Issue by jlg1203** [mleibman/SlickGrid#401]
This issue occur if you have SlickGrid under a JQuery UI Accordion.
1) In the slickgrid that is underneath an opened accordion section, move the vertical scrollbar of the slickgrid table at the middle or somewhere at the bottom of all the records.
2) You collapse the accordion section
3) When you open the accordion section, the slickgrid table displays a white space but when you move the vertical scrollbar then the slickgrid table displays fine again.

I am not sure if this is because of the collapse/expand that SlickGrid reference to what is shown is getting distorted?

**Comment by Domt301** [comment]
Did you ever find a way to resolve this issue? I am dealing with the same thing at the moment.

**Comment by jlg1203** [comment]
Unfortunately not. I can't remember if I just did a hard refresh that reloaded slickgrid on expand of the accordion.

**Comment by Domt301** [comment]
Yup ended up having to do the same thing. "window.location.reload();" its not great and i hope i can find a jquery magic trick to fix it but it works.

**Comment by 6pac** [comment]
Just a comment - if you could put together a test page as a jsfiddle, that would help me diagnose and fix the problem. Just a self contained page with minimal elements to reproduce the issue. I'll add it as a unit test to my repo once I've fixed it.

**Comment by 6pac** [comment]
A workaround for this issue has been added to the 'alternative master' repo.
Repo: https://github.com/6pac/SlickGrid
Latest release: https://github.com/6pac/SlickGrid/releases
Blog post: http://low-bandwidth.blogspot.com.au/2015/06/slickgrid-with-jqueryui-and-bootstrap.html

The latest release has added examples for Bootstrap and the jQueryUI Accordion that may help provide a standardised page for testing, and demonstrate a known good configuration.

The Accordion demo includes workaround code for this issue, commented out by default. Unfortunately, I couldn't come up with a fix.
